tests/test_templater.py

```diff
@@ -97,6 +97,31 @@ def test_set_template_yaml_config():
     os.environ['how_many'] = 'all'
 
 
+    # Make sure the output file matches the expected output
+    with tempfile.TemporaryDirectory(dir='.') as tmp_dir:
+        out_file = f'{tmp_dir}/test_render_from_yaml.nml'
+
+        args = [
+             '-i', input_file,
+             '-c', config_file,
+             '-o', out_file,
+             ]
+
+        templater.set_template(args)
+
+        assert filecmp.cmp(expected_file, out_file)
+
+def test_set_template_yaml_config_model_configure():
+    '''Tests that the templater will work as expected for a simple model_configure
+    file. '''
+
+    input_file = os.path.join(uwtools_file_base,
+                              "fixtures/model_configure.sample.IN")
+    config_file = os.path.join(uwtools_file_base,
+                               "fixtures/model_configure.values.yaml")
+    expected_file = os.path.join(uwtools_file_base,
+                                 "fixtures/model_configure.sample")
+
```

Comment:
Welp, that's an interesting way to do this diff. I added this last test, which is very similar to the previous one. I'd have expected it to just highlight the added lines at the end.